test/user-query-test.ts

```diff
@@ -0,0 +1,155 @@
+import { url } from "../src/setup-server";
+import { expect } from "chai";
+import { prisma } from "../src/setup-db";
+import { hashPassword } from "../src/graphql/resolvers";
+import { generateToken } from "../src/graphql/helpers/login-handlers";
+import { createUserMutation, getUserByID } from "./test-queries";
+import axios from "axios";
+
+describe("User Query Test", function () {
+  it("Found a user", async function () {
+    const hashedPassword = await hashPassword("123abc");
+
+    const userDB = await prisma.user.create({
+      data: {
+        name: "User Test",
+        email: "test@example.com",
+        password: hashedPassword,
+        birthDate: "01-01-2000"
+      }
+    });
+
+    const token = generateToken({ id: userDB.id, email: userDB.email }, false);
+    const userQueryResponse = await axios.post(
+      url,
+      {
+        query: getUserByID,
+        variables: {
+          id: userDB.id
+        }
+      },
+      {
+        headers: {
+          Authorization: token
+        }
+      }
+    );
+
+    const expectedUserDB = {
+      id: userDB?.id,
+      name: userDB?.name,
+      email: userDB?.email,
+      birthDate: userDB?.birthDate
+    };
```

Comment:
```suggestion
    const expectedUserDB = userDB && {
      id: userDB.id,
      name: userDB.name,
      email: userDB.email,
      birthDate: userDB.birthDate
    };
```

Só uma técnica pra ele fazer a verificação da existência desse userDb uma vez só em vez de fazer 4x.